### ISSUE TITLE
After `pip install groundwork-sphinx-theme` and updating config.py, theme not found

### ISSUE THREAD

**Issue by dyerrington** [useblocks/groundwork-sphinx-theme#4]
I've tried a few other sphinx themes with success but I found that after these steps, the `make html` command produces a theme not found type error:

1. `pip install groundwork-sphinx-theme`
1. nano conf.py
1. Updating `html_theme = 'groundwork'`

This error is produced:

```bash
make html
Running Sphinx v2.2.0
loading pickled environment... done

Theme error:
no theme named 'groundwork' found (missing theme.conf?)
make: *** [html] Error 2
```
Any suggestions?

**Comment by nnnewb** [comment]
just got same problem

**Comment by yoshihikoueno** [comment]
same problem here

**Comment by danwos** [comment]
Solved with version 0.1.1.
Thanks for reporting.
